default.nix

```diff
@@ -0,0 +1,53 @@
+{ pkgs ? import <nixpkgs> {}
+, useLLVMLibcxx ? false
+, ... }:
+
+with pkgs;
+
+pkgs.llvmPackages.stdenv.mkDerivation rec {
+  name = "xeus-cling";
+  src = ./.;
+
+  nativeBuildInputs = with pkgs; [
+    cmake
+    makeWrapper
+  ];
+
+  buildInputs = with pkgs; [
+    xeus
+    xeus-zmq
+    cppzmq
+    cling.unwrapped
```

Comment:
The default `cling` package is meant for use by end-users directly, so it only provides the `/bin/cling` script which feeds arguments to the unwrapped version. See my comment on `cxxFlags` below for why it needs to be wrapped.